### ISSUE TITLE
Provide a Series navigation to the Catalogue page

### ISSUE THREAD

**Issue by ajnyga** [pkp/pkp-lib#6978]
**Describe the problem you would like to solve**
Currently the only route to Series archive is through the Browse block plugin which can be very hidden especially in mobile layouts. We need additional routes to Series archives through the Catalogue page.

**Describe the solution you'd like**
Theme option to Default theme that allows you to list the available publisher Series. (mock-up from Nate)
![catalog-by-series](https://user-images.githubusercontent.com/16347527/115705172-e648da00-a374-11eb-93bc-67501b03b4eb.png)

**Comment by ajnyga** [comment]
OMP: https://github.com/pkp/omp/pull/966

**Comment by NateWr** [comment]
Merged to `main`. Thanks!

**Comment by NateWr** [comment]
@asmecher I'm running into a SQL error with the GROUP BY in [this query](https://github.com/pkp/omp/blame/main/classes/press/SeriesDAO.inc.php#L285-L293):

```
Syntax error or access violation: 1055 Expression #10 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'omp.p.publication_id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
```

I can see that the tests passed so I'm wondering if this is a configuration problem that I should fix or maybe something that's not caught by our tests?

**Comment by asmecher** [comment]
That's MySQL, I presume? What version?

**Comment by NateWr** [comment]
mysql  Ver 8.0.25-0ubuntu0.20.04.1 for Linux on x86_64 ((Ubuntu))

**Comment by asmecher** [comment]
@NateWr, I think this can be fixed by changing the
```
SELECT *
```
...to...
```
SELECT s.*
```
...but your version of MySQL is clearly pickier around this than Antti-Jussi's. If this fixes it on your machine, it's safe to commit for all. Can you check?

**Comment by NateWr** [comment]
You're a SQL genious, @asmecher! Fixed it up fine and I'll roll this fix into my PRs for #5328.
